clusterloader2/pkg/prometheus/manifests/prometheus-rules.yaml

```diff
@@ -63,6 +63,15 @@ spec:
       record: probes:dns_lookup_latency:histogram_quantile
       labels:
         quantile: "0.50"
+    - expr: |
+        probes_dns_propagation_count
+      record: probes:dns_propagation_count
+    - expr: |
+        probes_dns_propagation_seconds
+      record: probes:dns_propagation_seconds
```

Comment:
Do we need these recording rules if they are essentially copies?

Reply:
Removed